### PR TITLE
Specify that licenses found can be expressions.

### PR DESCRIPTION
**Pull request by zvr**
In all cases of Package, File, or Snippet, the information about
licenses found can be License Expressions, instead of single licenses.

Signed-off-by: Alexios Zavras <github@zvr.gr>

### CODE REVIEW

**Review by kestewart — APPROVED** [review]
Thanks Alexios,  not sure it was necessary to condense out the line spacing, but not worth redoing.   Edits look good to me.

**Comment by zvr** [comment]
@kestewart it's not "condensing the line spacing", it's "transforming paragraphs into a list". :-P

Now all cases have a list of three alternatives (expressions, `NONE`, `NOASSERTION`), with a second-level list following, specifying the conditions for the latter case.

File: chapters/package-information.md
Comment (seabass-labrax) [inline chapters/package-information.md:923]:
```suggestion
* A valid SPDX License Expression as defined in [Annex D](SPDX-license-expressions.md);
```

Would prefer if this was a bigger link target

File: chapters/snippet-information.md
Comment (seabass-labrax) [inline chapters/snippet-information.md:232]:
```suggestion
* A valid SPDX License Expression as defined in [Annex D](SPDX-license-expressions.md).
```

File: chapters/snippet-information.md
Comment (seabass-labrax) [inline chapters/snippet-information.md:297]:
```suggestion
* A valid SPDX License Expression as defined in [Annex D](SPDX-license-expressions.md).
```

**Review by seabass-labrax — CHANGES_REQUESTED** [review]

File: chapters/snippet-information.md
Comment (kestewart) [reply chapters/snippet-information.md:297]: @seabass-labrax - I think this is a "nice to have" rather than must fix before merging.   We're in the stage right now of trying to get this release out.   This sort of thing, can be fixed by PR later, rather than blocking a merge now.

File: chapters/snippet-information.md
Comment (kestewart) [reply chapters/snippet-information.md:232]: Not sure this should be blocking the merge change.

**Comment by goneall** [comment]
@zvr - If you're still online and want to add a commit to fix the formatting suggested by @seabass-labrax , I'll wait about 30 minutes to merge (10:30 AM Pacific time).  If I don't hear back, I'll merge this as is and we can create a separate PR to address the formatting issues.

**Comment by goneall** [comment]
> @zvr - If you're still online and want to add a commit to fix the formatting suggested by @seabass-labrax , I'll wait about 30 minutes to merge (10:30 AM Pacific time). If I don't hear back, I'll merge this as is and we can create a separate PR to address the formatting issues.

Going a head with the merge.  @seabass-labrax If you don't mind creating a PR to fix the formatting suggested above.

**Pull request merged**
